openedx/features/enterprise_support/utils.py

```diff
@@ -32,6 +33,11 @@ def get_data_consent_share_cache_key(user_id, course_id):
 
     return get_cache_key(type='data_sharing_consent_needed', user_id=user_id, course_id=course_id)
 
+def get_is_enterprise_cache_key(user_id):
+    """
+        Returns cache key for the enterprise learner validation method needed against user_id.
+    """
+    return get_cache_key(type='is_enterprise_learner', user_id=user_id)
```

Comment:
Nit: the pep8 linter is gonna complain about only 1 blank line before and after this function.